src/Maker/MakeController.php

```diff
@@ -91,6 +99,11 @@ public function configureDependencies(DependencyBuilder $dependencies)
             Annotation::class,
             'annotations'
         );
+
+        $dependencies->addClassDependency(
+            TwigBundle::class,
+            'twig-bundle'
+        );
```

Comment:
It's not a dependency. IF Twig is installed, we generate a template. If not, we don't.